src/parseTools.js

```diff
@@ -1052,13 +1052,20 @@ function hasExportedFunction(func) {
   return WASM_EXPORTS.has(_asmjsDemangle(func));
 }
 
+function documentI64Param(name) {
+  if (WASM_BIGINT) {
+    return `'/** @param {!BigInt} ${name}_bigint */'`;
```

Comment:
what error does this prevent?

Reply:
Lots of this stuff:

```
/tmp/emscripten_temp_337t15lk/hello_world.js:14230:30: WARNING - [JSC_BAD_TYPES_FOR_BINARY_OPERATION] operator & cannot be applied to ? and bigint
  14230|       var offset_low = Number(offset_bigint & BigInt(0xffffffff)) | 0, offset_high = Number(offset_bigint >> BigInt(32)) | 0;
                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Unless closure knows that `offset_bigint` is for sure a BigInt it won't let you `&` with a `BigInt`

Reply:
Huh... closure is pretty strict then, more than I'd expect...